src/main/java/uk/gov/companieshouse/api/strikeoffobjections/email/KafkaEmailClient.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.companieshouse.api.strikeoffobjections.email;
+
+import org.apache.avro.Schema;
+import org.apache.kafka.clients.producer.RecordMetadata;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.api.strikeoffobjections.model.model.EmailContent;
+import uk.gov.companieshouse.kafka.message.Message;
+import uk.gov.companieshouse.kafka.producer.CHKafkaProducer;
+import uk.gov.companieshouse.service.ServiceException;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
```

Comment:
The sonar report is claiming this to be an unused import

Reply:
Thought I'd got them all removed this one.